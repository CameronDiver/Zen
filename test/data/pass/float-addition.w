let a = 3.141;
let b = 3;
printf("%.3f\n", a+b);
let c = a + b;
printf("%.3f\n", c + 3.0);

