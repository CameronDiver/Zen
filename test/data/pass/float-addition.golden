6.141
9.141
